API-strategie-modulen/API-strategie-mod-geo/crs.md

```diff
@@ -87,7 +87,7 @@ The guiding principles for CRS support:
 - The default CRS, CRS84, is listed first in the list of supported CRSs in the API; if the consumer does not specify the CRS it is assumed it uses the default.
 - Coordinate reference systems API strategy: request/response in RD; ETRS89; CRS84; Pseudo  Mercator;
 - Consider no-regret: record in multiple much-requested CRSs instead of on-the-fly transformation;
-- Use RDNAPTRANS™ 2018 to transform RD/Amersfoort to ETRS89 (correction grid);
+- Use the latest version of RDNAPTRANS™ to transform RD to ETRS89 (correction grid);
```

Comment:
Ik zou hier toch maar meteen "ETRS89 (correction grid)" vervangen door: "ETRF2000 (including correction grid for the highest accuracy)."
Als we dat nu niet doen en het issue sluiten lopen we het risico dat we het bij andere issues vergeten mee te nemen.
We kunnen beter bij het verwerken van een ander issue tot de conclusie komen dat het op deze plek al is opgelost.

Reply:
Ik zou liever ETRS89 laten staan. Zie ook discussie in https://github.com/Geonovum/KP-APIs/issues/507